test/post.ts

```diff
@@ -262,12 +262,8 @@ test('the `form` payload is not touched', withServer, async (t, server, got) =>
 	});
 });
 
-test.failing('DELETE method sends plain objects as JSON', withServer, async (t, server, got) => {
-	server.delete('/', (request, response) => {
-		// Not using streams here to avoid this unhandled rejection:
-		// Error [ERR_STREAM_PREMATURE_CLOSE]: Premature close
-		response.json(request.body);
-	});
+test('DELETE method sends plain objects as JSON', withServer, async (t, server, got) => {
+	server.delete('/', defaultEndpoint);
```

Comment:
Can you undo this change? This needs to be (not necessarily in this PR) fixed, not covered up.

Reply:
I can undo it, but it's going to continue to fail. My test wasn't written correctly in the first place.

To fix the test independently, there are a couple options:

1. Use the streaming `defaultEndpoint` and attempt to catch the "premature close" error to avoid an unhandled rejection
2. Use a different test server configuration for this test that supports JSON body parsing

Would you prefer a separate PR that can be merged prior to this one?

Reply:
> Would you prefer a separate PR that can be merged prior to this one?

Can be a separate PR. I want to get out a new release now.